cmd/kthena-controller-manager/main.go

```diff
@@ -237,3 +244,70 @@ func waitForCertsReady(keyFile, CertFile string) bool {
 		time.Sleep(waitInterval)
 	}
 }
+
+func parseControllers(controllers string) map[string]bool {
```

Comment:
I would sugest first separate controllers, by enabled and disabled.

Then calculate the eventual enabled = enabled - disabled